signxml/xades/__init__.py

```diff
@@ -601,6 +573,37 @@ def _generate_xades(self, options_struct):
           up     := UnsignedProperties,             c. 4.3.3
             usp  := UnignedSignatureProperties,     c. 4.3.6
             udop := UnignedDataObjectProperties,    c. 4.3.7
+
+        Electronic signature forms elements
```

Comment:
"Electronic signature _grammar_  (I guess that's what it is) by mode"

Reply:
term used https://www.etsi.org/deliver/etsi_ts/101900_101999/101903/01.04.02_60/ts_101903v010402p.pdf at 4.4

Reply:
I'm ok with that, just those `?` look a little grammar-ish. But yep, in xml, that is probably not a common usage of a word...